src/scopes/grants/startDate.js

```diff
@@ -1,26 +1,100 @@
 import { Op } from 'sequelize';
-import { withinDateRange, compareDate } from '../utils';
+import { compareDate } from '../utils';
 
-export function beforeGrantStartDate(date) {
+export function beforeStartDate(date) {
   return {
     [Op.and]: {
       [Op.or]: compareDate(date, 'startDate', Op.lt),
     },
   };
 }
 
-export function afterGrantStartDate(date) {
+export function afterStartDate(date) {
   return {
     [Op.and]: {
       [Op.or]: compareDate(date, 'startDate', Op.gt),
     },
   };
 }
 
-export function withinGrantStartDates(dates) {
+export function withinStartDates(dates) {
```

Comment:
Should this be renamed to something like `activeWithinDates`? Something letting the user of the function know this scope looks for active grants between `dates`?